addons/calendar/static/src/xml/base_calendar.xml

```diff
@@ -28,6 +28,13 @@
         </t>
     </t>
 
+    <t t-name="Calendar.attendee.calendar-box">
+        <div t-att-style="typeof color === 'string' ? ('background-color:' + color) + ';' : ''" t-attf-class="#{record.is_highlighted ? 'o_event_hightlight' : ''} #{typeof color === 'number' ? _.str.sprintf('o_calendar_color_%s', color) : 'o_calendar_color_1'}">
+            <span class="fc-time"/>
```

Comment:
Not sure I understand your changes in this commit.
You override the "_eventRender" method to be able to call a custom template that just adds a "fc-time" span that is not used anywhere else in the commit?

What am I missing?

Reply:
The custom template is to add css classes depending of the attendee_status
#{record.attendee_status === 'needsAction' ? 'o_attendee_status_not_accepted': ''} #{record.attendee_status === 'declined' ? 'o_attendee_status_declined': ''

Reply:
the template is copy from addons/web/static/src/xml/web_calendar.xml
I just added the classes that I need to the default template

    <t t-name="calendar-box">
        <div t-att-style="typeof color === 'string' ? ('background-color:' + color) + ';' : ''" t-attf-class="#{record.is_highlighted ? 'o_event_hightlight' : ''} #{typeof color === 'number' ? _.str.sprintf('o_calendar_color_%s', color) : 'o_calendar_color_1'}">
            <span class="fc-time"/>
            <div class="o_event_title" t-esc="record.display_name"/>
        </div>
    </t>

Reply:
test